### PR TITLE
Enhancements to invoicing 

### PR DESCRIPTION
**Pull request by aanil**

### CODE REVIEW

**Review by sofiahag — APPROVED** [review]
Looks good!

You could create a dictionary instead as you said. I guess you would create it as:

`let badge_colour = {	
        "Aborted": "bg-danger",
        "Closed": "bg-success",
        "Reception Control": "bg-secondary",
        "Ongoing": "bg-info",
        "Pending": "bg-info",
      }`
      
And access the value by `badge_colour[value['project_status']]`? 

Or something along those lines 🙂

**Review by sofiahag — APPROVED** [review]
OK, looks good to me! 👍

File: run_dir/static/js/invoicing.js [hunk 1]
```diff
@@ -18,9 +19,11 @@ function load_invoicing_table() {
       let tbl_row = $('<tr>')
       let checkbox_col = '<input class="form-check-input invoice_checkbox align-middle" type="checkbox" value="'+key+'" id="sel_invoices_'+key+'">'
       tbl_row.append($('<td class="mw-5 text-center">').html(checkbox_col))
-      let project_row = '<a class="text-decoration-none" href="/project/'+key+'">'+key+'</a>'
+      let badge_colour = { "Aborted": "bg-danger", "Closed": "bg-success", "Reception Control": "bg-secondary", "Ongoing": "bg-info", "Pending": "bg-info"}
+      let project_row = '<a class="text-decoration-none" href="/project/'+key+'">'+key+', '+value['project_name']+'</a>'
       project_row += '<button type="button" id='+key+' class="btn btn-sm btn-outline-dark view_invoice_btn float-right px-3" data-toggle="modal" data-target="#displayInvoiceModal">View</button>'
       tbl_row.append($('<td>').html(project_row))
+      tbl_row.append($('<td>').html('<span class="badge '+badge_colour[value['project_status']]+'">'+value['project_status']+'</span>'));
```
Comment (alneberg) [inline run_dir/static/js/invoicing.js:26]:
```suggestion
      tbl_row.append($('<td>').html('<h4 class="mb-0"><span class="badge '+badge_colour[value['project_status']]+'">'+value['project_status']+'</span></h4>'));
```

File: run_dir/static/js/pricing_quote.js [hunk 2]
```diff
@@ -447,15 +487,15 @@ app.component('v-pricing-quote', {
...
               <div class="row" v-if="this.saved_agreement_data['saved_agreements']">
                 <h4 v-if="this.saved_agreement_data">Generated Agreements</h4>
                 <div class="col ml-2">
                   <template v-for="(agreement, timestamp) in this.saved_agreement_data['saved_agreements']" :key="timestamp">
                         <div class="form-check m-2">
-                          <input class="form-check-input" type="radio" name="saved_agreements_radio" :id="timestamp" :value="timestamp">
+                          <input class="form-check-input" type="radio" name="saved_agreements_radio" :id="timestamp" :value="timestamp" :checked="this.saved_agreement_data['signed']===timestamp">
                           <label class="form-check-label" :for="timestamp">
-                          {{ timestamp_to_date(timestamp) }}, {{ agreement['created_by']}}
+                          <div v-bind:class="{ 'fw-bold' : timestamp === this.loaded_timestamp}"> {{ timestamp_to_date(timestamp) }}, {{ agreement['created_by']}} </div>
```
Comment (alneberg) [inline run_dir/static/js/pricing_quote.js:498]: Nice!

**Review by alneberg — APPROVED** [review]
Great work!

**Pull request merged**